client/my-sites/sites/index.jsx

```diff
@@ -207,11 +208,13 @@ class Sites extends Component {
 				path = translate( 'Add-ons' );
 				break;
 		}
+		// We don't have a translated path, capitalize the provided path.
+		const capitalize = initialPath === path;
 
 		return translate( 'Select a site to open {{strong}}%(path)s{{/strong}}', {
 			args: { path },
 			components: {
-				strong: <strong />,
+				strong: <strong className={ clsx( { 'strong--capitalize': capitalize } ) } />,
```

Comment:
This works well to preserve the capitalization of the translation, but a lot of the words are pretty generic and have capitalization [built-in in the translation](https://translate.wordpress.com/projects/wpcom/-all-translated/1072/). One way to solve it would be to add context to each of the cases, or we could just create unique strings for each case, like `Select a site to open {{strong}}import{{/strong}}`, which would have the benefit of better sounding originals since we don't have to follow the pattern anymore.

Reply:
@dlind1 I agree, the translation quality would be improved by adding context/creating separate strings. 

This PR updates the capitalization rule to ensure we preserve the original capitalization when we have a custom string / vs just the path. 

I can tackle the strings in a separate diff.

Reply:
Sounds good. Current phrases are quite awkward and should be refactored.